libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1689,5 +1689,25 @@
         "value": "Create",
         "comment": "",
         "limit": 0
+    },
+    "styling.editor.dialog.name": {
+        "value": "Name",
+        "comment": "",
+        "limit": 0
+    },
+    "styling.editor.dialog.definition": {
+        "value": "Definition",
+        "comment": "",
+        "limit": 0
+    },
+    "styling.editor.dialog.examples": {
+        "value": "Examples",
+        "comment": "",
+        "limit": 0
+    },
+    "styling.editor.dialog.example.paste": {
+        "value": "Paste",
```

Comment:
This one might need a comment for the translators as it may have multiple meanings if I'm not mistaken.